agent/api/task.go

```diff
@@ -39,8 +39,8 @@ const (
 	// variable containers' config, which will be used by the AWS SDK to fetch
 	// credentials.
 	awsSDKCredentialsRelativeURIPathEnvironmentVariableName = "AWS_CONTAINER_CREDENTIALS_RELATIVE_URI"
-	// pauseContainerName is the internal name for the pause container
-	pauseContainerName = "~internal~ecs~pause"
+	// PauseContainerName is the internal name for the pause container
```

Comment:
Per https://github.com/aws/amazon-ecs-agent/pull/765, there's a proper type for the internal container now.  Can you make this private and use the type instead for comparisons?